### ISSUE TITLE
a question about Warmup dataloader mode

### ISSUE THREAD

**Issue by jiangyangbo** [LiJunnan1992/DivideMix#25]
in the process of warmup,  the mode is 'all', it means it will take all the train data for training , and  the train data include  both 
 the labeled data  and  the unlabeled data, it may have some influence on the input of GMM,  the unlabeled data with it's labled are trained in the process of warmup,  what i say is right?

**Comment by LiJunnan1992** [comment]
hi, thanks for your interest!

Yes during warmup all data is used for training. The network is expected to learn something useful from all training data, such that it can produce high loss for noisy data when the GMM starts.
